upsolver-api/seed.js

```diff
@@ -0,0 +1,47 @@
+import fs from "fs";
+import path from "path";
+import { fileURLToPath } from "url";
+import { User, Team, Role } from "./models/index.js";
+import { sequelize } from "./database.js";
+import bcrypt from "bcrypt";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const userData = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, "./seeders/users.json"), "utf8")
+);
+const teamData = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, "./seeders/teams.json"), "utf8")
+);
+const roleData = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, "./seeders/roles.json"), "utf8")
+);
+
+userData.forEach(async (user) => {
+  user.password = await bcrypt.hash(user.password, 10);
+});
+
+const seedDatabase = async () => {
+  try {
+    await sequelize.query("DROP TABLE IF EXISTS users");
+    await sequelize.query("DROP TABLE IF EXISTS roles");
+    await sequelize.query("DROP TABLE IF EXISTS teams");
```

Comment:
this is looks like a good opportunity to await all these promises concurrently, as all of the queries are independent of each other. Look into `Promise.all` for that: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

Reply:
Fixed in: https://github.com/cdamezcua/Upsolver/pull/22